src/test/java/org/pesmypetcare/webservice/dao/medalmanager/UserMedalDaoTest.java

```diff
@@ -58,7 +57,11 @@
 
     @BeforeEach
     public void setUp() {
-        userMedalEntity = new UserMedalEntity("Walker", 1.0, 2.0, new ArrayList<>());
+        medal = new Medal();
+        medal.setDescription("You have to walk a lot!");
+        medal.setLevels(new ArrayList<>(Arrays.asList(5., 10., 25., 50., 100.)));
+        medal.setMedalIconPath(new byte[] {10, 55, 67, 89, 103, 116});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['10' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5630992)